### DISCUSSION TITLE
[Feature] Merge albums

### DISCUSSION THREAD

**Discussion by lscorcia** [immich-app/immich#12676] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have imported a bunch of folders into Immich using the CLI. I used the --album switch to transform those folders into separate albums. However, some of the folders were just "partitions" of the pictures, and I'd rather merge them with their parent folder album. A merge album feature in the UI would be useful in this case.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#3190
